### PR TITLE
SCMOD-5073: Unused dependencies by internal modules removed

### PR DESCRIPTION
**Pull request by thilagasanthosh**
https://autjira.microfocus.com/browse/SCMOD-5073

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -429,35 +313,15 @@ limitations under the License.
...
-            <dependency>
-                <groupId>commons-configuration</groupId>
-                <artifactId>commons-configuration</artifactId>
-                <version>1.10</version>
-            </dependency>
-            <dependency>
-                <groupId>commons-io</groupId>
-                <artifactId>commons-io</artifactId>
-                <version>2.4</version>
+                <groupId>com.brsanthu</groupId>
```
Comment (michael-bryson) [inline pom.xml]: This is not in the correct place alphabetically.

File: util-liquibase-installer/pom.xml [hunk 2]
```diff
@@ -56,7 +54,6 @@
         <dependency>
```
Comment (michael-bryson) [inline util-liquibase-installer/pom.xml:53]: There is a version listed here on line 51, this should be moved to the parent pom.

**Review by michael-bryson — CHANGES_REQUESTED** [review]

File: util-liquibase-installer/pom.xml (hunk 2 above)
Comment (thilagasanthosh) [reply util-liquibase-installer/pom.xml:53]: @dermot-hardy , Is there any specific purpose for overriding postgresql version  in 
https://github.com/CAFapi/caf-common/blob/SCMOD-5073-r2.0.0/util-liquibase-installer/pom.xml#L51?
Please confirm.

Comment (dermot-hardy) [reply util-liquibase-installer/pom.xml:53]: I don't think so.  It should probably just be moved to the parent POM.

**Comment by michael-bryson** [comment]
Is there a developer build for this?

**Comment by thilagasanthosh** [comment]
Developer build:
http://sou-jenkins2.hpeswlab.net/job/CAFapi/view/Developer/job/CAFapi~caf-common~SCMOD-5073-r2.0.0~CI/

**Comment by michael-bryson** [comment]
enforceCorrectDependencies flag is not set on the parent pom.

**Comment by michael-bryson** [comment]
Why are there no check-dependencies results in the build log?
http://sou-jenkins2.hpeswlab.net/job/CAFapi/view/Developer/job/CAFapi~caf-common~SCMOD-5073-r2.0.0~CI/2/consoleFull

**Comment by thilagasanthosh** [comment]
@dermot-hardy , 
Could you confirm whether this branch should have dependency issues fix as like 1.17 version caf-common?

**Comment by dermot-hardy** [comment]
Ideally yes

**Comment by thilagasanthosh** [comment]
@michael-bryson ,

New Jira created for adding check-dependencies to caf-common 2.0.0 branch.
https://autjira.microfocus.com/browse/SCMOD-5742
As of now only cleanup required for part of 5073 story.

**Review by michael-bryson — APPROVED** [review]

**Pull request merged**